src/process_utility.c

```diff
@@ -239,13 +239,33 @@ check_continuous_agg_alter_table_allowed(Hypertable *ht, AlterTableStmt *stmt)
 	}
 }
 
+static bool
+hypertable_has_compressed_chunks(Hypertable *ht)
```

Comment:
Isn't there already a function for this: `ts_chunk_exists_with_compression`? Or is it doing something else. Seems a bit unnecessary to build all chunks just to check for the occurrence of  one.